packages/telegram-cloud-storage-stamper/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# @turnkey/telegram-cloud-storage-stamper
+
+## 0.1.0
+
+Initial release
```

Comment:
if you have a changeset I think you don't need a changelog? It'll automatically be generated when someone calls `pnpm run changeset version`